BLSQ/iaso
# POLIO-1519: Adding campaign types columns on calendar and map

Adding campaign types columns on calendar and map pages

Related JIRA tickets : POLIO-1519

## Self proofreading checklist



## Changes

Added columns to static columns on calendar page and campaign list page

## How to test

Open campaigns page and check that type columns is displaying correct infos, play with the sort.
Make the same thing on calendar page

## Print screen / video

![Screenshot 2024-03-14 at 10 19 25](https://github.com/BLSQ/iaso/assets/12494624/cb1aab58-9e98-477c-90a8-1e5fb0527ff0)
![Screenshot 2024-03-14 at 10 19 20](https://github.com/BLSQ/iaso/assets/12494624/8e2d26d2-2000-43bc-b090-4f932be024ef)
